musicrs/util/db.py

```diff
@@ -1,56 +1,41 @@
 """ Database utility module. """
 
 import pyodbc
+from sqlalchemy import create_engine
+from sqlalchemy.engine.url import URL
+from sqlalchemy.orm import sessionmaker
 
-from musicrs.util.object import merge
+from musicrs.model.base import Base
+from musicrs.settings import DATABASE_CONN
 
-from musicrs.api.logging import get_logger
 
-# Database connection types
-PG: str = "pg"
-MSSQL: str = "mssql"
-MYSQL: str = "mysql"
 
-# Drivers
-DRIVER_PG = "{PostgreSQL Unicode}"
-DRIVER_MSSQL = "{ODBC Driver 17 for SQL Server}"
+def create_db_engine():
+    """ Create database connection engine"""
+    database_url = URL(**DATABASE_CONN)
+    db_engine = create_engine(database_url)
+    return db_engine
 
-# Defaults
-DEFAULTS = {}
-DEFAULTS[PG] = {"driver": DRIVER_PG, "port": "5432"}
-DEFAULTS[MSSQL] = {"driver": DRIVER_MSSQL, "port": "1433"}
-DEFAULTS[MYSQL] = {"driver": "{MySQL ODBC 8.0 Driver}", "port": "3306"}
 
-# Connection strings
-CONN_STR: str = ";".join(
-    [
-        "DRIVER={driver}",
-        "SERVER={host}",
-        "PORT={port}",
-        "DATABASE={database}",
-        "UID={username}",
-        "PWD={password}",
-    ]
-)
+def create_db_session():
```

Comment:
missing space above

Reply:
Spaces or new line? There are two spaces above. You can see that line 18 and 19 are empty

Reply:
oh i meant the new line.